src/main/java/uk/gov/hmcts/darts/audio/mapper/GetAudioRequestResponseMapper.java

```diff
@@ -25,9 +25,9 @@ public GetAudioRequestResponseV1 mapToAudioRequestSummary(EnhancedMediaRequestIn
         response.setMediaRequestStatus(MediaRequestStatus.fromValue(enhancedMediaRequestInfo.getMediaRequestStatus().toString()));
         response.setLastAccessedTs(transformedMedia.getLastAccessed());
         response.setOutputFilename(transformedMedia.getOutputFilename());
-        String outputFilename = transformedMedia.getOutputFormat();
-        if (outputFilename != null) {
-            response.setOutputFormat(AudioRequestOutputFormat.fromValue(outputFilename));
+        uk.gov.hmcts.darts.audio.enums.AudioRequestOutputFormat outputFormat = transformedMedia.getOutputFormat();
+        if (outputFormat != null) {
+            response.setOutputFormat(AudioRequestOutputFormat.fromValue(outputFormat.toString().toUpperCase()));
```

Comment:
Do you need the `toUpperCase` here - I thought it would just take the enum name which is already in caps?

Reply:
switched it back to lowercase so needed here